addon/templates/components/popover-on-element.hbs

```diff
@@ -1,3 +1,12 @@
-{{yield (hash
-	hide=(action "hide")
-)}}
+{{#if _shouldRender}}
```

Comment:
Correct. But let me clarify the first bullet point...

- `_shouldRender` will be `false` by default if `enableLazyRendering` is `true`
  * `_shouldRender` will be `true` by default if `enableLazyRendering=true` **and** `isShown=true`